crates/ruff_python_ast/src/source_code/indexer.rs

```diff
@@ -59,25 +64,34 @@ impl Indexer {
 
             match tok {
                 Tok::Comment(..) => {
-                    commented_lines.push(*range);
+                    comment_ranges.push(*range);
                 }
                 Tok::Newline | Tok::NonLogicalNewline => {
                     line_start = range.end();
                 }
                 Tok::String {
-                    triple_quoted: true,
+                    triple_quoted,
+                    kind,
                     ..
-                } => string_ranges.push(*range),
+                } => {
+                    if *triple_quoted {
+                        triple_quoted_string_ranges.push(*range);
+                    }
+                    if matches!(kind, StringKind::FString | StringKind::RawFString) {
```

Comment:
Can an fstring be triple quoted? If not, use an `else if` or use two separate `match` arms, one for triple-quoted strings and another for fstrings.

Reply:
It can! We actually don't need to use those anywhere since we only care about single-quoted f-strings... But it seems strange to omit them.